### ISSUE TITLE
Upload de grandes arquivos

### ISSUE THREAD

**Issue by josircg** [larhud/nuvem#8]
Criar nova view /transcribe que implemente a leitura de arquivos grandes (1Gb!!!) com progress bar

**Comment by fabiofsilva** [comment]
Descreva com mais detalhes, pois não conheço o projeto.
Como será a interface para o usuário? Somente uma página com um campo de upload de arquivo? Onde devo salvar o arquivo?

**Comment by josircg** [comment]
Exatamente! Você pode pegar o mesmo template da home: http://nuvem.farmi.pro.br/
removendo todos os outros campos.

Salve o arquivo em disco, usando a mesma estrutura de Documento.

**Comment by josircg** [comment]
Como fica se 2 pessoas fizerem o upload ao mesmo tempo nessa estrutura de arquivo temporário?

**Comment by fabiofsilva** [comment]
Cada arquivo gera um id e a view retorna para o cliente no padrão abaixo:
{
    "upload_id": "5230ec1f59d1485d9d7974b853802e31",
    "offset": 10000,
    "expires": "2013-07-18T17:56:22.186Z"
}Toda vez que faz o POST para o servidor, tem que passar o id retornado e a próxima parte do arquivo.
    Em terça-feira, 4 de julho de 2023 às 07:13:46 BRT, Josir Cardoso Gomes ***@***.***> escreveu:  
 
 


Como fica se 2 pessoas fizerem o upload ao mesmo tempo nessa estrutura de arquivo temporário?

**Comment by josircg** [comment]
Ficou 10! 

1) Como você resolveu o problema da migrate que ficou de fora?
2) A rotina só deve aceitar wav, ogg, mp3, webm e mp4, mov, avi e flv.

**Comment by fabiofsilva** [comment]
1) Eu gerei a migração no meu PC. Não adicionei, porque não sei se tem alguém trabalhando nisso.2) Eu aproveitei o form de upload de documento que só permite txt e pdf. Esquece a validação que está lá?    Em quarta-feira, 5 de julho de 2023 às 10:26:06 BRT, Josir Cardoso Gomes ***@***.***> escreveu:  
 
 


Ficou 10!
   
   - Como você resolveu o problema da migrate que ficou de fora?
   - A rotina só deve aceitar wav, ogg, mp3, webm e mp4, mov, avi e flv.
